### ISSUE TITLE
A few questions to ask

### ISSUE THREAD

**Issue by AmyGrrl** [xerpi/ds4vita#48]
I'm having a few problems while using this plugin. Mainly with the left analog stick. I'm using a Vita 1000 on 3.65 Enso and the CUH-ZCT2G model controller. I had no problem getting the controller to sync. Showed up as Wireless Controller. Then I rebooted the system and now it shows Dualshock 4 Wireless Controller. Great!

However the left analog stick doesn't seem to work correctly in a lot of games. Where I will get choppy/glitchy movement with my character. My character will only walk slowly instead of a fast walk. I read in a previous issue that this newer model controller may not be properly supported. Was this issue ever fix? Would getting the older model of controller fix these issues with the left analog stick? 

Also I know there are a bunch of test builds floating around that may fix various issues. Currently using the one that has a MD5 Checksum of 39F19102CC21FEAF3191AEF425EB71D4. This version does seem to work a little better. Is there any newer test builds then this one that may fix the problems I am having.

I'm kind of sad that this great and amazing plugin seems to be dead now and the bugs/problems with it won't be fixed now. Would have been great if the unused L/R buttons could be mapped to the read touch pad and could be programmed to do different touch functions for different games.

**Comment by xerpi** [comment]
Hi, have you given this version a try? https://github.com/xerpi/ds4vita/issues/43#issuecomment-431850355

**Comment by AmyGrrl** [comment]
Yes. I am already using the version. Which is why I posted the md5 hash so you would know which version I was using. Still have left analog stick issues with many games. Do I need to get the older model of controller??

**Comment by xerpi** [comment]
I see. I don't think getting the older model would change anything since this plugin is a hack anyways. You can try to connect it to your PC and see if the joysticks are well centered.

**Comment by AmyGrrl** [comment]
Thought I would post an update this. I finally found my PSTV this week in a box. Hooked it up and tried using the controller with that. I was still having the same problems with the controller. Which I thought was odd since I was using the official driver and not this plugin. So I had a friend bring over their PS4 controller yesterday. Everything worked perfectly fine with that controller on my PSTV and with this plugin on my Vita. So looks like the left thumbstick on the used PS4 controller I bought is completely messed up. I'm going to take it back to the store today and trade it back in and just buy a new controller.

Lesson learned. Don't buy used controllers! You can close this issue now if you want.
